taichi/transforms/scalarize.cpp

```diff
@@ -270,12 +278,12 @@ class Scalarize : public BasicStmtVisitor {
     auto const &formats = stmt->formats;
     std::vector<std::variant<Stmt *, std::string>> new_contents;
     // Sparse mapping between formatted expr and its specifier
```

Comment:
```suggestion
```
This comment can be removed now~ I apologize for my mistake on using the wrong data structure!